appveyor.yml

```diff
@@ -26,5 +31,19 @@ after_build:
 
     $wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "DesktopClrTestResults.xml"))
 
+on_success:
+- ps: >-
+    if ($env:APPVEYOR_REPO_TAG -eq "true")
+    {
+        nuget push .\build\Castle.Core.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.log4netIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.NLogIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.SerilogIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+    }
+
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
```

Comment:
Yes, this is a way of aliasing or naming the artifacts for use later on. We have removed all the providers so it does not make much sense on it's own here like this. 

An example of it being used is here: https://github.com/cryosharp/fortress/blob/f5c7e3b686946f6869ba8b3c0f45e0c2211d651a/appveyor.yml#L44

Want me to remove it?